lighthouse-core/gather/driver.js

```diff
@@ -570,9 +572,15 @@ class Driver {
       });
   }
 
-  beginTrace() {
+  /**
+   * @param {{additionalTraceCategories: string=}=} flags
+   */
+  beginTrace(flags) {
```

Comment:
true, but would still need the additionalTraceCategories check so just gets rid of `flags &&` in favor of ` = {additionalTraceCategories: ''}` gotta save those bytes you know ;)